resources/assets/styles/common/_global.css

```
@@ -112,7 +112,7 @@ a {
   text-decoration: none;
 }
 
-a:not([href*="platform.coop"]):not([href^="#"]):not([href^="/"])::after {
+a:not([href*="platform.coop"]):not([href*="localhost:10008"]):not([href^="#"]):not([href^="/"])::after {
```

Removing the port number will make this work for other localhost environments ([reference](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors)). 

```suggestion
a:not([href*="platform.coop"]):not([href*="localhost"]):not([href^="#"]):not([href^="/"])::after {
```